pkg/clients/ovirt/impl.go

```diff
@@ -69,6 +69,11 @@ func (is *ovirtClient) CreateVMByMachine(
 		if providerSpec.MemoryMB > 0 {
 			vmBuilder.Memory(int64(math.Pow(2, 20)) * int64(providerSpec.MemoryMB))
```

Comment:
Please change `int64(math.Pow(2, 20)` to a constant like in the TF provider PR

Reply:
done